content/PHPDataObjects/DataObjects.q.md

```diff
@@ -523,6 +523,110 @@ In `student.php`, which statement correctly describes `echo $row['student_id'].$
 :::
 
 
+:::
+
+/// type=REPL, readonly=true, init=[commands/DataObjects/UpdateBirthDate.sql], filename=[connection.php,student.php]
```

Comment:
Please check this one here... I think we have to be consistent with our arrangement. Which should come first `init` or `filename`?